torch/nn/parallel/data_parallel.py

```diff
@@ -60,9 +91,13 @@ def __init__(self, module, device_ids=None, output_device=None, dim=0):
         self.module = module
         self.device_ids = device_ids
         self.output_device = output_device
+        torch.cuda._lazy_call(self.check_cuda_balance)
```

Comment:
We could also use a lambda here.

```python
torch.cuda._lazy_call(lambda: _check_balance(self.device_ids))
```

Reply:
I don't think we have to use _lazy_call here. Just call `_check_balance(self.device_ids)` directly